utils/markdown/renderer.tsx

```diff
@@ -275,7 +275,7 @@ export default class Renderer extends marked.Renderer {
         return output;
     }
 
-    public listitem(text: string, bullet = '') {
+    public listitem(text: string, bullet = '') { // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
Are we removing this `eslint-disable-line` in a future PR? Or is this a special case?

Reply:
This is a special case. The type definitions provided for `marked` don't match what we use (either because we're based off an old version of marked or because of our fork, I can't remember), and some of the overridden Renderers need that extra parameter.